packages/pds/src/storage/disk-blobstore.ts

```diff
@@ -112,9 +112,13 @@ export class DiskBlobStore implements BlobStore {
   }
 
   async getStream(cid: CID): Promise<stream.Readable> {
+    const path = this.getStoredPath(cid)
+    const exists = await fileExists(path)
+    if (!exists) {
+      throw new BlobNotFoundError()
+    }
     try {
-      const handle = await fs.open(this.getStoredPath(cid), 'r')
-      return handle.createReadStream()
+      return fsSync.createReadStream(path)
     } catch (err) {
       if (isErrnoException(err) && err.code === 'ENOENT') {
         throw new BlobNotFoundError()
```

Comment:
This error can't really occur here anymore, I don't think—I would expect it to instead be emitted on the returned stream.  That could technically still happen if the file is removed between the `fileExists()` check above and when the file stream is created, though I don't think that is especially likely or a major deal.

Reply:
yup yup good call 👍 

fixed both of these up & changed my approach just a bit

avoiding open handles in general, just wanna create a read stream & let node handle closing it

Reply:
ohhh got it you're probably right

ah well, this feels a bit more explicit to me so i'll stick with it

Reply:
I think that sounds good 💎